includes/sql-schema/update.php

```diff
@@ -59,6 +59,9 @@
     return;
 }
 
+// Set Database Character set and Collation
+dbQuery('ALTER DATABASE ? CHARACTER SET utf8 COLLATE utf8_unicode_ci;', array(array($config['db_name'])));
```

Comment:
Yes, the second array removes the surrounding quotes.

Otherwise, the query is `ALTER DATABASE 'librenms' CHARACTER SET utf8 COLLATE utf8_unicode_ci;`